docs/package.json

```diff
@@ -32,6 +32,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-expo": "^9.2.2",
     "babel-preset-react-native": "^4.0.1",
+    "canvas": "^2.11.2",
```

Comment:
Why didn't you try `https://www.npmjs.com/package/@vercel/og`? 🤔

Reply:
canvas seemed easier